### PR TITLE
CSVS-42 Application config was fixed

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

File: src/main/scala/com/bwsw/cloudstack/vault/server/util/Constants.scala [hunk 1]
```diff
@@ -25,11 +25,16 @@ object ConfigLiterals {
...
 
-  val tokenPeriod            = s"$applicationDomain.tokenPeriod"
-  val accountsVaultBasicPath = s"$applicationDomain.accountsVaultBasicPath"
-  val vmsVaultBasicPath      = s"$applicationDomain.vmsVaultBasicPath"
   val tagNamePrefix          = s"$applicationDomain.tagNamePrefix"
 
+  val vaultUrl               = s"$vaultDomain.url"
+  val vaultRootToken         = s"$vaultDomain.rootToken"
+  val vaultRetryDelay        = s"$vaultDomain.retryDelay"
+  val tokenPeriod            = s"$vaultDomain.tokenPeriod"
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/vault/server/util/Constants.scala]: make the variable names in one uniform style

**Review by placccebo — DISMISSED** [review]

File: src/main/scala/com/bwsw/cloudstack/vault/server/util/Constants.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/vault/server/util/Constants.scala]: ok

File: src/main/scala/com/bwsw/cloudstack/vault/server/util/Constants.scala [hunk 2]
```diff
@@ -30,9 +30,9 @@ object ConfigLiterals {
   val vaultUrl               = s"$vaultDomain.url"
   val vaultRootToken         = s"$vaultDomain.rootToken"
   val vaultRetryDelay        = s"$vaultDomain.retryDelay"
-  val tokenPeriod            = s"$vaultDomain.tokenPeriod"
-  val accountsVaultBasicPath = s"$vaultDomain.accountsBasicPath"
-  val vmsVaultBasicPath      = s"$vaultDomain.vmsBasicPath"
+  val vaultTokenPeriod       = s"$vaultDomain.tokenPeriod"
+  val vaultAccountsBasicPath = s"$vaultDomain.accountsBasicPath"
+  val vaultVMsBasicPath      = s"$vaultDomain.vmsBasicPath"
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/vault/server/util/Constants.scala]: vaultVMsBasicPath -> vaultVmsBasicPath

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/main/scala/com/bwsw/cloudstack/vault/server/util/Constants.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/vault/server/util/Constants.scala]: ok

**Review by placccebo — APPROVED** [review]

**Pull request merged**